### ISSUE TITLE
Add testcase for multidc scaleout with 90% utilization in parallel

### ISSUE THREAD

**Issue by pehala** [scylladb/scylla-cluster-tests#9310]
At 90% storage usage, scale out each DC by adding a single node to each of them in parallel.

**Comment by cezarmoise** [comment]
## Results

<details><summary>Without throttle r/w during scaleout</summary>
<p>

Initial Cluster: 2 DC with 3 x i4i.large each
Write to 70%; Sleep 30 min
Write to 90%; Sleep 30 min
Writes have RF=3
Add 1 x i4i.large to each DC
Sleep 30 minutes

![Image](https://github.com/user-attachments/assets/74b8cb32-8e72-44ed-8621-69c90d9e2b4e)


</p>
</details>

**Comment by pehala** [comment]
Merged into https://github.com/scylladb/scylla-cluster-tests/issues/9157
